src/libparodus_private.h

```diff
@@ -380,13 +395,121 @@ typedef enum {
 } __libpd_err_t;
 
 
+
+typedef struct {
+	int err_detail;	// err detail of last api call
+			// the codes are defined here in libparodus_private.h
+	int oserr;	// os err of last api call
+} extra_err_info_t;
+
+
+/**
+ * Initialize the parodus wrp interface
+ *
+ * @param instance pointer to receive instance object that must be provided
+ *   to all subsequent API calls.
+ * @param cfg configuration information: service_name must be provided,
```

Comment:
Add @param function header comments for each func describing the use of  err_info and also add comments not to be used in production in the beginning of this header file